### PR TITLE
Info frontend

### PR DESCRIPTION
**Pull request by LiamK**
Changes for adding multiple Deliberations (Info nodes).

### CODE REVIEW

File: ui/src/components/Deliberations/components/DeliberationList/DeliberationList.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+const AnchorUrl = styled.a`
+  display: block;
+  text-decoration: none;
+  :hover { text-decoration: none; }
+  :active { text-decoration: none; }
+  :visited { text-decoration: none; }
+`;
+
+// <AnchorUrl href={url} key={nodeId} target="_blank">
```
Comment (erikfrisk) [inline ui/src/components/Deliberations/components/DeliberationList/DeliberationList.js]: Remove commented out code

File: ui/src/components/Deliberations/components/DeliberationList/DeliberationList.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+// </AnchorUrl>
+
+//
+
+const Deliberations = ({ deliberations, showRemove }) => (
+  <StyledListGroup>
```
Comment (erikfrisk) [inline ui/src/components/Deliberations/components/DeliberationList/DeliberationList.js]: Component should have same name as file

File: package.json [hunk 3]
```diff
@@ -23,6 +23,8 @@
   },
   "dependencies": {
     "babel-eslint": "^8.0.3",
-    "concurrently": "^3.5.1"
+    "concurrently": "^3.5.1",
+    "normalize-url": "^4.0.0",
```
[the remark this answers is not in the document]
Comment (LiamK) [reply package.json]: I moved normalize-url to ui/package.json   Unfortunately, that leads to an issue neither I nor Gustav were able to resolve.  The normalize-url package was working okay when I included it in the top-level package.json file, but in ui/package.json if fails on import with:
```
./node_modules/normalize-url/index.js
Module parse failed: Unexpected token (22:2)
You may need an appropriate loader to handle this file type.
| removeDirectoryIndex: false,
| sortQueryParameters: true,
| ...options
| };
|
```
Seems like something with webpack.  Can you take a look at it?

I have commented out the related import and usage in AddDeliberation.js.
I have left the dependency reference in ui/package.json.

[the remark this answers is not in the document]
Comment (LiamK) [reply package.json]: node -v; npm -v
v8.9.2
6.4.1

**Pull request merged**